packages/playwright/README.md

```diff
@@ -198,6 +198,37 @@ axeResults = await locator.RunAxe(options);
 
 
 ```
+## Migrating from `Playwright.Axe` ([PlaywrightAxeDotnet](https://github.com/IsaacWalker/PlaywrightAxeDotnet))
+
+This project acts as a drop-in replacement for most of the functionality from `Playwright.Axe`. ([PlaywrightAxeDotnet](https://github.com/IsaacWalker/PlaywrightAxeDotnet)). To migrate:
+
+1. Update your test `.csproj` file's `<PackageReference>` for `Playwright.Axe` to `Deque.AxeCore.Playwright`
+1. Add a new `<PackageReference>` to `Deque.AxeCore.Commons` at the same version number as `Deque.AxeCore.Playwright`
+1. Update all `using Playwright.Axe;` statements in your tests to `using Deque.AxeCore.Playwright;` and/or `using Deque.AxeCore.Commons;`
+
+There are a few minor breaking changes which won't impact most `Playwright.Axe` users, but which may require updates if you use certain advanced features:
+1. The `.Target` and `.XPath` properties of `AxeResultNode` or `AxeResultRelatedNode` are now strongly-typed `AxeSelector` objects. Most `Selenium.Axe` users do not refer to these properties explicitly, but if your tests do, you will probably want to do so via their `ToString()` representations.
```

Comment:
A few things with this item: 
* Looks like this part still references `Selenium.Axe`.
* Previously in the Playwright package, `AxeResultNode` was actually called `AxeNodeResult` and `AxeResultRelatedNode` was `AxeRelatedNode`
* There were a couple of other changes to the typings used in the Playwright package, including renaming, adding, and removing properties. A high-level summary of the typings changes can be found in the PR description of https://github.com/dequelabs/axe-core-nuget/pull/63 so that could be a good starting point

Reply:
Made an attempt at covering the migration changes as well in the latest commit! Let me know what else we need to add/change

Reply:
Thanks! Left you some comments offline